.github/workflows/build_and_test_ansi.yml

```diff
@@ -17,18 +17,18 @@
 # under the License.
 #
 
-name: ANSI SQL mode test
+name: "Build and test (ANSI)"
 
 on:
   push:
     branches:
       - '**'
 
 jobs:
-  ansi_sql_test:
+  call-build-and-test:
```

Comment:
I borrowed "call" word from the main docs: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-12